TNAPI/login.py

```diff
@@ -30,4 +34,29 @@ def login(user, password, path=r"geckodriver.exe"):
     time.sleep(3)
     sid = browser.get_cookie("connect.sid")["value"]
     browser.close()
-    return sid
\ No newline at end of file
+    return sid
+else:
```

Comment:
Else is not used as an alternative when try fails.  See other suggestions for how it should be done.